lib/services/ios-project-service.ts

```diff
@@ -262,6 +262,13 @@ class IOSProjectService extends projectServiceBaseLib.PlatformProjectServiceBase
 				this.savePbxProj(project).wait();
 			}
 			
+			if(this.$fs.exists(this.projectPodFilePath).wait() && this.$fs.getFsStats(this.projectPodFilePath).wait().mtime < this.$fs.getFsStats(this.$projectData.projectFilePath).wait().mtime) {
+				try {
+					this.$childProcess.exec("pod install", { cwd: this.platformData.projectRoot }).wait();
+				} catch(e) {
+					this.$errors.failWithoutHelp("CocoaPods are not installed. Run `sudo gem install cocoapods` and try again.");
```

Comment:
Shouldn't we display the error as well? For example CocoaPods might be installed but the command `pod install` to fail for some reason?